src/AuthenticatedAPIClient/tests/AuthenticatedAPIClient.test.jsx

```diff
@@ -330,10 +358,26 @@ describe('Token refresh failures', () => {
         return client[method](mockApiEndpointPath).catch(() => {
           expectSingleCallToJwtTokenRefresh();
           expectNoCallToCsrfTokenFetch();
-          expect(logError).toHaveBeenCalledWith(
-            'frontend-auth: Request failed with status code 403',
-            expect.any(Object),
-          );
+          expectLogErrorToHaveBeenCalledWithMessage('[frontend-auth] HTTP Client Error: 403 http://auth.example.com/api/refreshToken undefined');
+          expectLogout();
+        });
+      });
+    });
+  });
+
+  describe('The refresh request fails due to a timeout', () => {
+    beforeEach(() => {
+      setJwtCookieTo(null);
+      accessTokenAxiosMock.onPost().timeout();
+    });
+
+    ['get', 'options', 'post', 'put', 'patch', 'delete'].forEach((method) => {
+      it(`${method.toUpperCase()}: throws an error and redirects`, () => {
+        expect.hasAssertions();
+        return client[method](mockApiEndpointPath).catch(() => {
+          expectSingleCallToJwtTokenRefresh();
+          expectNoCallToCsrfTokenFetch();
+          expectLogErrorToHaveBeenCalledWithMessage('[frontend-auth] HTTP Client Error: timeout of 0ms exceeded post http://auth.example.com/api/refreshToken');
```

Comment:
Do we want to check custom metrics as well?

Reply:
Done